### ISSUE TITLE
Android drop-in target 33 - SecurityException

### ISSUE THREAD

**Issue by slawekkrol** [Adyen/adyen-android#788]
**Describe the bug**
We setup 3DS 2 on android, now when we tried to run drop-in on cards with 3DS error occurs. Version of SDK: 4.7.1

```
09:31:43
CO.DropInServiceKt
dispatching DropInServiceResult
09:31:43
CO.DropInActivityKt
handleDropInServiceResult - Action
09:31:43
CO.ActionHandler
handleAction - threeDS2
09:31:43
CO.ActionHandler
handleAction - loaded a new component - Adyen3DS2Component
09:31:43
CO.Adyen3DS2Component
identifyShopper - submitFingerprintAutomatically: true
09:31:43
CO.Adyen3DS2Component
initialize 3DS2 SDK
09:31:43
CO.Adyen3DS2Component
Unexpected uncaught 3DS2 Exception
09:31:43
CO.Adyen3DS2Component
java.lang.SecurityException: Settings key: <data_roaming> is only readable to apps with targetSdkVersion lower than or equal to: 32
09:31:43
CO.Adyen3DS2Component
	at android.provider.Settings$NameValueCache.getStringForUser(Settings.java:3081)
09:31:43
CO.Adyen3DS2Component
	at android.provider.Settings$Global.getStringForUser(Settings.java:15854)
09:31:43
CO.Adyen3DS2Component
	at android.provider.Settings$Global.getString(Settings.java:15837)
09:31:43
CO.Adyen3DS2Component
	at android.provider.Settings$Global.getInt(Settings.java:16078)
09:31:43
CO.Adyen3DS2Component
	at atd.s.b.b(Unknown Source:13)
09:31:43
CO.Adyen3DS2Component
	at atd.s.b.a(Unknown Source:0)
09:31:43
CO.Adyen3DS2Component
	at atd.h.b.a(Unknown Source:62)
09:31:43
CO.Adyen3DS2Component
	at atd.h.b.<init>(Unknown Source:34)
09:31:43
CO.Adyen3DS2Component
	at atd.h.b.a(Unknown Source:6)
09:31:43
CO.Adyen3DS2Component
	at com.adyen.threeds2.internal.g.initialize(Unknown Source:45)
09:31:43
CO.Adyen3DS2Component
	at com.adyen.checkout.adyen3ds2.Adyen3DS2Component$identifyShopper$1.invokeSuspend(Unknown Source:56)
09:31:43
CO.Adyen3DS2Component
	at f9.a.resumeWith(Unknown Source:11)
09:31:43
CO.Adyen3DS2Component
	at gc.r0.run(Unknown Source:88)
09:31:43
CO.Adyen3DS2Component
	at nc.a.A(Unknown Source:0)
09:31:43
CO.Adyen3DS2Component
	at nc.a$c.c(Unknown Source:14)
09:31:43
CO.Adyen3DS2Component
	at nc.a$c.m(Unknown Source:28)
09:31:43
CO.Adyen3DS2Component
	at nc.a$c.run(Unknown Source:0)
```

**To Reproduce**
Set target and compileSDK to 33
Setup 3ds 2 on adyen side
Try to use test or real card with 3DS 2

**Expected behavior**
3DS triggers

**Smartphone (please complete the following information):**
 - Device: Pixel 5
 - Version Android API 33

**Comment by jreij** [comment]
hi @slawekkrol, thanks for reaching out! We are aware of this issue and are planning to release a new version soon which includes a fix. we'll update this ticket when we have the fix.

**Comment by jreij** [comment]
Issue fixed, will be included in the `4.8.0` release.
